azure_functions_worker/constants.py

```diff
@@ -18,6 +18,7 @@
 
 # Platform Environment Variables
 AZURE_WEBJOBS_SCRIPT_ROOT = "AzureWebJobsScriptRoot"
+CONTAINER_NAME = "CONTAINER_NAME"
```

Comment:
Can we have this constant in dependency.py itself? This isn't being used globally.